site/posts/intake-esm-2020-3-16.ipynb

```diff
@@ -0,0 +1,483 @@
+    "\n",
+    "\n",
+    "On GitHub: https://github.com/NCAR/intake-esm\n",
+    "\n",
+    "Documentation: https://intake-esm.readthedocs.io/\n",
+    "\n",
+    "<!-- TEASER_END -->\n",
+    "\n",
+    "## Installation\n",
+    "\n",
+    "Intake-esm can be installed from PyPI with pip:\n",
+    "\n",
+    "```bash\n",
+    "python -m pip intall intake-esm --upgrade\n",
+    "```\n",
+    "\n",
+    "It is also available from conda-forge channel for conda isntallations:\n",
+    "\n",
+    "```bash\n",
+    "conda install -c conda-forge intake-esm\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## New Features\n",
+    "\n",
+    "\n",
+    "### Enhanced search: enforce query criteria via `require_all_on` argument\n",
+    "\n",
+    "By default intake-esm's `search()` method returns entries that fulfill **any of the criteria** specified in the query. Today intake-esm can return entries that fulfill **all query criteria** when the user supplies the `require_all_on` argument. `require_all_on` consits of attributes/dataframe columns to use when enforcing the query criteria.\n",
+    "\n",
+    "`require_all_on argument` is best explained with examples:\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "pangeo-cmip6-ESM Collection with 235624 entries:\n",
+       "\t> 15 activity_id(s)\n",
+       "\n",
+       "\t> 32 institution_id(s)\n",
+       "\n",
+       "\t> 69 source_id(s)\n",
+       "\n",
+       "\t> 101 experiment_id(s)\n",
+       "\n",
+       "\t> 135 member_id(s)\n",
+       "\n",
+       "\t> 29 table_id(s)\n",
+       "\n",
+       "\t> 313 variable_id(s)\n",
+       "\n",
+       "\t> 10 grid_label(s)\n",
+       "\n",
+       "\t> 235624 zstore(s)\n",
+       "\n",
+       "\t> 60 dcpp_init_year(s)"
+      ]
+     },
+     "execution_count": 1,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Open collection for CMIP6 data hosted on Google Storage\n",
+    "import intake\n",
+    "\n",
+    "url = \"https://git.io/JvP9r\"\n",
+    "col = intake.open_esm_datastore(url)\n",
+    "col"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define our query\n",
+    "\n",
+    "query = dict(\n",
+    "    variable_id=[\"thetao\", \"o2\"],\n",
+    "    experiment_id=[\"historical\", \"ssp245\", \"ssp585\"],\n",
+    "    table_id=[\"Omon\"],\n",
+    "    source_id=[\"ACCESS-ESM1-5\", \"AWI-CM-1-1-MR\", \"FGOALS-f3-L\"],\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "- Search for assets that fulfill our query without `require_all_on`"
```

Comment:
Don't use a bullet here.  It indents too much for just 1 line between the code blocks, making it harder to read.

And expand on what the reader is about to see, something like:

```suggestion
    "Now, let's use this `query` to search for all assets in the collection that satisfy *any combination* of these requests (i.e., with `require_all_on=None`, which is the default):"
```